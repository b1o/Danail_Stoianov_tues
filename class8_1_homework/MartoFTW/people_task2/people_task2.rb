require "csv"

out_arr = []

	CSV.foreach(ARGV[0]) do |row|

		row[3] = row[3].to_i

		if row[1].to_f >= ARGV[1].to_f
			out_arr << row 
		end
	end

	out_arr = out_arr.sort_by{|a,b,c,d| d}
	out_arr.reverse!

	CSV.open("people_task2_result.csv","w") do |csv|
		out_arr.each do |elem|
			csv << elem
		end
	end
	
