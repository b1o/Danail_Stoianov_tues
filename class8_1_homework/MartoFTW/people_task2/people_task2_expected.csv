Slavi Georgiev,1.89,87 kg,43
Derek Storm,1.95,93 kg,35
Vasil Simeonov,1.85,85 kg,33
Bruce Lee,1.80,70 kg,30
Lebron James,2.15,120 kg,29
Petyr Petrov,1.81,90 kg,20
Fluffy Notfat,1.75,100 kg,19
